art/src/node.rs

```diff
@@ -613,12 +618,7 @@ impl<V> Default for NodeBody4<V> {
     fn default() -> Self {
         Self {
             keys: [KEY_INVALID; 4],
-            children: [
-                NodeBox::null(),
-                NodeBox::null(),
-                NodeBox::null(),
-                NodeBox::null(),
-            ],
+            children: arr![NodeBox::null();4],
```

Comment:
It seems it's not `cargo fmt`ed.  Please run it.

Reply:
ok, i see
i remove the comment and 
run `cargo fmt` to rerange code